.travis.yml

```diff
@@ -135,13 +136,10 @@ jobs:
         - master
   - stage: smoke
     script: npm run test:integration:remote
+  - stage: update translations
+    script: npm run i18n:push
+    if: branch == develop AND type == cron
 stages:
 - test
 - name: smoke
   if: type != pull_request
```

Comment:
I think this is going to run the smoke test on every `cron` job unless we change it to `type NOT IN (cron, pull_request`

Reply:
OK, take a look now